raven/src/intTest/java/org/opencadc/raven/NegotiationTest.java

```diff
@@ -835,4 +837,63 @@ public Object run() throws Exception {
             Assert.fail("unexpected exception: " + e);
         }
     }
+
+    @Test
+    public void testPreauthURL() throws Exception {
+        List<Protocol> requested = new ArrayList<>();
```

Comment:
probably don't need lines 843-846 and 875 below - that's for testing something else.

Reply:
C&P error. Fixed